hudi-common/src/test/java/org/apache/hudi/common/minicluster/HdfsTestService.java

```diff
@@ -66,7 +66,7 @@ public Configuration getHadoopConf() {
   }
 
   public MiniDFSCluster start(boolean format) throws IOException {
-    Preconditions.checkState(workDir != null, "The work dir must be set before starting cluster.");
+    Objects.requireNonNull(workDir, "The work dir must be set before starting cluster.");
```

Comment:
How about adding `checkState` to ValidationUtils?

Reply:
For this null check, its unnecessary - there is a checkState in ValidationUtils to check other boolean conditions. But null check, its fine to use Bojects.checkNotNull()